.github/workflows/sdk_build.yml

```diff
@@ -73,37 +90,42 @@ jobs:
           python scripts/gha/install_prereqs_desktop.py
           cd ..
 
-      # Set up the requirements, and install Unity
-      - uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: 2.6
-      - name: Install Unity installer (U3D)
-        shell: bash
-        run: gem install u3d
       - name: Install python deps
         shell: bash
         run: |
           pip install -r scripts/gha/requirements.txt
-      - name: Install Unity
-        shell: bash
-        run: |
-          python scripts/gha/unity_installer.py --install --platforms ${{ matrix.unity_platform_name }} --version ${{ matrix.unity_version }}
-      - name: Setup Unity path
-        shell: bash
+
+      - name: Install OpenSSL (Windows)
+        if: startsWith(matrix.os, 'windows')
         run: |
-          echo "UNITY_ROOT_DIR=${{ matrix.unity_install_dir }}" >> $GITHUB_ENV
+          choco install openssl -r
 
       - name: Install OpenSSL (Linux)
         if: startsWith(matrix.os, 'ubuntu')
         run: |
           sudo apt install openssl
+
       - name: Install OpenSSL (macOS)
         if: startsWith(matrix.os, 'macos')
         run: |
           brew install openssl
           # brew won't overwrite MacOS system default OpenSSL, so force it here.
           echo "OPENSSL_ROOT_DIR=$(brew --prefix openssl --installed)" >> $GITHUB_ENV
 
+      - name: Install Unity
+        shell: bash
+        run: |
+          u3d available -u ${{ matrix.unity_version }} -p
```

Comment:
Is this cmd for testing only?

Reply:
It's for information, and I don't see downside to put it here.